docs/applications/key-manager.md

```diff
@@ -0,0 +1,15 @@
+---
+id: key-manager
+title: Key Manager
+---
+
+
+#### Key Manager
+
+The key manager module's goal is to encapusulate all private key interactions with the [wallet](wallet.md) project.
+
+As of this writing, the wallet just delegates storage of the encrypted mnemonic seed to the key manager project. Over the log run, we want to make it so that the wallet project needs to communicate with the key-manager to access private keys.
+
+This means that ALL SIGNING should be done inside of the key-manager, and private keys should not be leaked outside of the key manager.
```

Comment:
nit "should not be leaked outside of" -> "should not leave" (they shouldn't be leaked anywhere ;)